### PR TITLE
fix: data table fixes

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?

Fixes datatable being broken on large orgs with lots of attributes
https://app.campsite.com/cal/posts/juz8fo0i1tmu#comment-g2tsifhimbch

Also adds a sticky header and sticky select and member field

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

**Review by alishaz-polymath — DISMISSED** [review]
LGTM

File: packages/features/users/components/UserTable/UserListTable.tsx [hunk 1]
```diff
@@ -186,6 +186,10 @@ export function UserListTable() {
         id: "select",
         enableHiding: false,
         enableSorting: false,
+        meta: {
+          sticky: true,
+          stickLeft: 0,
+        },
```
Comment (sean-brydon) [inline packages/features/users/components/UserTable/UserListTable.tsx:192]: Allows us to have sticky and offset the left column depending on what column is being used

Comment (alishaz-polymath) [reply packages/features/users/components/UserTable/UserListTable.tsx:192]: Quick Q: Would this affect RTL-LTR layout differences?

**Review by Udit-takkar — CHANGES_REQUESTED** [review]
type checks are failing

**Review by Udit-takkar — APPROVED** [review]

**Pull request merged**